### ISSUE TITLE
ARB_fragment_program support: allow benign respecification of precision hints

### ISSUE THREAD

**Issue by chrisforbes** [chrisforbes/mesa#34]
The spec says:

```
Only one precision control option may be specified by any given 
fragment program.  A fragment program that specifies both the
"ARB_precision_hint_fastest" and "ARB_precision_hint_nicest" program
options will fail to load.
```

It's not particularly meaningful to specify the same hint twice, but nVIDIA allows it, and so Unity3D games emit this kind of junk in some of their shaders.

**Comment by chrisforbes** [comment]
Fixed in ksp-fixes

**Comment by chrisforbes** [comment]
Fix is now in master, queued for 9.1.x
